homeassistant/components/onboarding/views.py

```diff
@@ -429,6 +430,115 @@ async def post(self, manager: BackupManager, request: web.Request) -> web.Respon
         return await self._post(request)
 
 
+def setup_cloud_views(hass: HomeAssistant, data: OnboardingStoreData) -> None:
+    """Set up the cloud views."""
+
+    # The cloud integration is imported locally to avoid cloud being imported by
+    # bootstrap.py and to avoid circular imports.
+
+    # pylint: disable-next=import-outside-toplevel
+    from homeassistant.components.cloud import http_api as cloud_http
+
+    # pylint: disable-next=import-outside-toplevel,hass-component-root-import
+    from homeassistant.components.cloud.const import DATA_CLOUD
+
+    class CloudOnboardingView(HomeAssistantView):
```

Comment:
As discussed on Discord, if we add more functionality to onboarding, we should consider adding an onboarding platform.